### ISSUE TITLE
Support configuration file for WAR deployment too

### ISSUE THREAD

**Issue by jcayzac** [plantuml/plantuml-server#169]
The JAR supports the `--config` option, could the WAR get the same capability please?

**Comment by HeinrichAD** [comment]
For documentation purpose:

The `--config` flag is an option of the deprecated jetty-runner, which is one possible way to deploy servlet/JSP container. It does not really have anything to do with the PlantUML Server itself.

For this reason, it becomes difficult/impossible to give the desired functionality to the packaged and bundled web application (war file, which stands for _Web application ARchive_) that can be deployed on _any_ servlet/JSP container.

In short, to get the functionality of the `--config` flag from the jetty-runner you need to check and configure your deployment system. E.g. tomcat or jetty server.

Edit: And if the `-config` flag from the core plantuml.jar is meant, there is the possibility to set the environment variable `PLANTUML_CONFIG_FILE`. See [README](https://github.com/plantuml/plantuml-server#how-to-set-plantuml-options).

**Comment by jcayzac** [comment]
> there is the possibility to set the environment variable `PLANTUML_CONFIG_FILE`. See [README](https://github.com/plantuml/plantuml-server#how-to-set-plantuml-options).

This was introduced in cf717eff0ce3dbac431501000eec52925f507986, more than a year after I created this issue, and solves it.
